lib/src/directions.dart

```diff
@@ -27,7 +27,7 @@ class GoogleMapsDirections extends GoogleWebService {
     TravelMode travelMode,
     List<Waypoint> waypoints,
     bool alternatives,
-    RouteType avoid,
+    List<RouteType> avoids,
```

Comment:
to avoid breaking change
can you keep the `RouteType avoid` parameter and add `@deprecated` annotation ?